index.bs

```diff
@@ -6604,10 +6607,10 @@ of user tracking:
     User agents may require the user to authorize access to databases
     before a site can use the feature.
 
-: Origin-tracking of stored data
+: Storage Key tracking of stored data
 ::
-    User agents may record the [=/origins=] of sites that contained content
-    from third-party origins that caused data to be stored.
+    User agents may record the [=/storage key=]s of sites that contained content
```

Comment:
You should be able to put the 's' inside the link; bikeshed is smart enough to handle (some) plural forms.